From 9e54868715f7785211ba6d1a5ccbea8fe9fc7c8d Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 17 May 2018 11:32:20 +0800
Subject: [PATCH 3815/5242] MLK-18331 ARM64: dts: freescale: imx8qm: update
 cpufreq set points

commit  0cf37ed9603c093eca7646f9fe3141fdd9f27ca7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update cpufreq set-points according to SCFW changes:

A53: add 1104MHz setpoint;
A72: add 1300MHz setpoint.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 5607348..ff63290 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -177,6 +177,7 @@
 		/* kHz    uV */
 		/* voltage is maintained by SCFW, so no need here */
 		1200000	   0
+		1104000	   0
 		900000	   0
 		600000	   0
 	>;
@@ -191,6 +192,7 @@
 		/* voltage is maintained by SCFW, so no need here */
 		1596000	   0
 		1464000	   0
+		1300000	   0
 		1056000	   0
 		600000     0
 	>;
-- 
1.7.9.5

